recipes-wigwag/maestro/maestro/rpi3/devicedb.template.conf

```diff
@@ -182,8 +182,8 @@ cloud:
     # historyID: "*.wigwag.io"
     # historyHost and historyPort may be ommitted. In this case historyHost and
     # historyPort are set to the normal cloud host and port
-    historyURI: {{ARCH_GW_SERVICES_URL}}/relay-history/history
-    alertsURI: {{ARCH_GW_SERVICES_URL}}/relay-alerts/alerts
+    historyURI: http://{{FOG_PROXY_ADDR}}/relay-history/history
```

Comment:
Double checking my understanding.

This, and other URLs, aren't https because this is all localhost, correct?

Reply:
Yes, thats correct.